api/envoy/admin/v3/config_dump.proto

```diff
@@ -381,3 +381,18 @@ message EndpointsConfigDump {
   // The dynamically loaded endpoint configs.
   repeated DynamicEndpointConfig dynamic_endpoint_configs = 3;
 }
+
+// Envoy's admin fills this message with init managers, which provides the information of their unready targets.
+message UnreadyTargetsConfigDumpList {
```

Comment:
Is this actually configuration? It's definitely implied by configuration when interpreted by Envoy, but I'm wondering if it belongs on a different endpoint. @envoyproxy/api-shepherds

Reply:
> Is this actually configuration? It's definitely implied by configuration when interpreted by Envoy, but I'm wondering if it belongs on a different endpoint. @envoyproxy/api-shepherds

I suppose it does not belong to an actual endpoint. This is just a protobuf message designed to show the unready targets in configs.

Reply:
I think it's kind of weird to include as part of config dump, but I'll defer to what other @envoyproxy/api-shepherds reckon. @mattklein123 as well as he did the original config dump.

Reply:
+1, can we spit this out somewhere else? This is not really config in the normal sense of the word. Maybe a new `/init_manager` or other endpoint?

Reply:
got it. I will move it somewhere else, and create a new init_dump.proto

Reply:
Sorry I'm late to this thread, I feel this should be part of config_dump, otherwise it will be hard to tell what config made the targets unready, querying two endpoints might end up in inconsistent (racy) dump.

Reply:
Isn't this a problem that applies to the admin endpoint in general? If we query `/config_dump` and `/clusters`, we might see inconsistent output. Same goes for `/listeners`. Would it make sense to try solve this through some mechanism for batching, e.g. an API endpoint, rather than trying to atomically group things?

Reply:
I feel like /config_dump or the potential is an inspection endpoint. 
The configs/targets are stale as soon as these init manager status are on the wire.

Reply:
rst has been updated.